models/accounts.js

```diff
@@ -0,0 +1,83 @@
+const mongoose = require('mongoose');
+
+const accountSchema = new mongoose.Schema({
+	title: {
+		type: String,
+		required: true,
+		unique: true
+	},
+	currency: {
+		type: String,
+		required: true
+	},
+	ammount: {
+		type: Number,
+		default: 0
+	},
+	income: {
```

Comment:
It's very bad idea to store incomes and expences in account. Move to another entity